### DISCUSSION TITLE
Jährliche Wetterlage

### DISCUSSION THREAD

**Discussion by OE3MNS** [Daveiano/weewx-wdc#214] [Q&A] [answered]
Hallo.
[Wo kann man die Parameter anpassen ?](https://www.wetter-kreuth.at/year-2023.html) 
Bei Sturmtage würde ich gern auf >60 km/h ändern.  

Vielen Dank. Schönes Wochenende. 
Lg Michi

**Comment by Daveiano** [comment, marked as answer]
Hallo Michi,

das ist bisher nicht konfigurierbar. Als Sturmtag gilt ein Tag wenn die Windgeschwindigkeit Beafort 8 überschritten hat (62km/h), das ist so definiert, siehe https://de.wikipedia.org/wiki/Sturmtag

Falls du das wirklich ändern möchstest, müsstest du zwei Stellen in weewx_wdc.py ändern (bei Fragen kann ich gerne unterstützen):

https://github.com/Daveiano/weewx-wdc/blob/3.x/bin/user/weewx_wdc.py#L2216
und 
https://github.com/Daveiano/weewx-wdc/blob/3.x/bin/user/weewx_wdc.py#L2317

Da diese "Klimatologischen Kenntage" als solche definiert sind, macht es eigentlich auch wenig Sinn das zu ändern, meine ich. 

Lg David

**Reply by OE3MNS** [reply]
Hallo David. 

Herzlichen Danke für deine Antwort. Gib jetzt natürlich eine sinn. [Klimatologische](https://de.wikipedia.org/wiki/Klimatologie) Kenntage! 

Lg Michi
